### PR TITLE
Added terminate for RabbitMQ Consumers

### PR DESCRIPTION
**Pull request by DmitryKK**

### CODE REVIEW

File: lib/message_queue/adapters/rabbitmq/consumer.ex [hunk 1]
```diff
@@ -67,6 +67,18 @@ defmodule MessageQueue.Adapters.RabbitMQ.Consumer do
         {:noreply, state, :hibernate}
       end
 
+      @impl true
+      def terminate(_, %{channel: channel} = _state) do
+        if not is_nil(channel) and Process.alive?(channel.pid) do
```
Comment (anronin) [inline lib/message_queue/adapters/rabbitmq/consumer.ex:72]:
```suggestion
        if is_pid(channel[:pid]) and Process.alive?(channel.pid) do
```

Comment (Judis) [reply lib/message_queue/adapters/rabbitmq/consumer.ex:72]: channel can be `nil` and you got error.

Comment (anronin) [reply lib/message_queue/adapters/rabbitmq/consumer.ex:72]: no, nil wil be returned, try `nil[:pid]`

Comment (DmitryKK) [reply lib/message_queue/adapters/rabbitmq/consumer.ex]: done

**Pull request merged**